Components/GridPostsTeaser/_style.scss

```diff
@@ -0,0 +1,148 @@
+[is='flynt-grid-posts-teaser'] {
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    padding: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
+  }
+
+  .preContent {
+    @include reset-content-margins;
+
+    margin-bottom: 4.375rem;
+    text-align: center;
+  }
+
+  .postContent {
+    margin-top: $container-padding-mobile + 10;
```

Comment:
I would vote to make this the same as the preContent margin-bottom and also align with having breakpoints (or rather: not having breakpoints).